sentry_sdk/tracing.py

```diff
@@ -280,10 +279,6 @@ def __enter__(self):
         # set as the implicit current context
         self._ctx_token = context.attach(ctx)
 
-        # get the new scope that was forked on context.attach
```

Comment:
this is now done universally in the context manager instead of here which would just affect the sentry created spans